src/main/java/ru/yandex/clickhouse/ClickHousePreparedStatementParameter.java

```diff
@@ -0,0 +1,49 @@
+package ru.yandex.clickhouse;
+
+import java.util.TimeZone;
+
+import ru.yandex.clickhouse.util.ClickHouseValueFormatter;
+
+public final class ClickHousePreparedStatementParameter {
+
+    private final String stringValue;
+    private final boolean quoteNeeded;
+
+    public static ClickHousePreparedStatementParameter fromObject(Object x,
+        TimeZone dateTimeZone, TimeZone dateTimeTimeZone)
+    {
+        if (x == null) {
+            return new ClickHousePreparedStatementParameter(null, false);
```

Comment:
1.) Let's make this object a static final field, and use it, in order to avoid extra allocations.
2.) Provide public static method ClickHousePreparedStatementParameter::null(), returning this field.
(use here and in PreparedStatement.setNull())